shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -299,6 +306,7 @@
   UIView* touchInterceptor = touch_interceptors_[view_id].get();
   touchInterceptor.layer.transform = CATransform3DIdentity;
   touchInterceptor.frame = frame;
+  touchInterceptor.alpha = 1;
```

Comment:
I think since we are adding more stuff here and might add even more later,  we can have a possible refactoring to make a separate method to do all of these: `ResetEmbeddedViewMutation(int view_id, EmbeddedViewParams params)`